### PR TITLE
Updated aREST switch doc for invert

### PR DESCRIPTION
**Pull request by w1ll1am23**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/14467

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/switch.arest.markdown [hunk 1]
```diff
@@ -46,6 +46,7 @@ Configuration variables:
 - **name** (*Optional*): Let you overwrite the name of the device. By default *name* from the device is used.
 - **pins** array (*Optional*): An array with all used pins.
   - **name** (*Required*): The name of the pin to use in the frontend.
+  - **invert** (*Optional*): If the logic for on/off should be inverted. 1 = off and 0 = on
```
Comment (syssi) [inline source/_components/switch.arest.markdown]: Please add the default value. The type (bool) would be nice, too. A `{% configuration %}` block would be awesome. :-)

Comment (frenck) [reply source/_components/switch.arest.markdown]: @w1ll1am23 @syssi 
1 = off? So true, is not to invert?

Comment (w1ll1am23) [reply source/_components/switch.arest.markdown]: No no, I was trying to call out a reference to the aREST code. (Arduino) 1 or HIGH is off and 0 or LOW is on.

Comment (frenck) [reply source/_components/switch.arest.markdown]: Aah check

@w1ll1am23, are you planning on changing/updating this PR, based on the feedback by @syssi ?

Comment (w1ll1am23) [reply source/_components/switch.arest.markdown]: @frenck yes, didn't get a chance yesterday. I'll try to get it done later today.

File: source/_components/switch.arest.markdown [hunk 2]
```diff
@@ -39,18 +42,39 @@ switch:
...
+  required: optional
+  type: map
+  keys:
+    name:
+      description: The name of the pin to use in the frontend.
+      required: true
+      type: string
+    invert:
+      description: If the logic for on/off should be inverted.
+      required: optional
```
Comment (syssi) [inline source/_components/switch.arest.markdown]: `required: false`. Is `optional` also valid?

Comment (frenck) [reply source/_components/switch.arest.markdown]: Well valid as in "Does it work?"; yes. Matching our standards: no.

**Review by frenck — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**